app/views/steps/alternatives/negotiation_tools/edit.en.html.erb

```diff
@@ -1,59 +1,56 @@
 <% title 'Negotiation tools and services' %>
+<% step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
+
+    <span class="govuk-caption-l">Alternative ways to reach an agreement</span>
+    <h1 class="govuk-heading-xl">Negotiation tools and services</h1>
+
+    <p class="govuk-body">
+      If you still communicate with the other person, and there are no concerns about domestic or child abuse, the
+      cheapest and easiest way to make arrangements is to negotiate with each other.
+    </p>
+    <p class="govuk-body">
+      There are free tools and services such as
+      <a href="https://www.splittingup-putkidsfirst.org.uk/home" class="govuk-link" rel="external" target="_blank">parenting plans</a>
```

Comment:
Let's change this link as per slack thread, as it is dead. We can use same or similar copy as in CAIT.

Reply:
Updated on 3b8855f0cc9d4bf0af7d55359d977440d16a9f5f